src/ui/public/vis_maps/visualizations/_map.js

```diff
@@ -60,10 +61,17 @@ export default function MapFactory(Private, tilemapSettings, Notifier) {
         fadeAnimation: false,
       };
 
+      let url;
       if (tilemapSettings.hasError()) {
-        notify.fatal(tilemapSettings.getError());
+        if (!previousErrors.includes(tilemapSettings.getError())) {
+          notify.warning(tilemapSettings.getError().message);
+          previousErrors.push(tilemapSettings.getError());
```

Comment:
Only displaying the error once for the duration of all instances of the TileMapMap means the user will only see the error once, until they re-open Kibana/refresh, right? I could see this being problematic if the user saw the error once, it started working again and they didn't close down kibana, and then a while later ran into the same issue.